src/tejashree/Assignment9.java

```diff
@@ -0,0 +1,68 @@
+package tejashree;
+
+public class Assignment9 {
+		int creditCount;
+		int debitCount;
+	    double currentBalance;
+		static int totalCreditCount;
+		static int totalDebitCount;
+		static int totalPrintBalCount;
+
+		void setCurrentBal(double currentBal){
+			currentBalance = currentBal;
+		}
+
+		void creditAmount(int creditAmount){
+			currentBalance = currentBalance + creditAmount;
+			creditCount++;
+			totalCreditCount++;
+		}
+
+		void debitAmount(int debitAmountt){
+			currentBalance = currentBalance - debitAmountt;
+			debitCount++;
+			totalDebitCount++;
+		}
+
+		void printBalance(){
+			System.out.println("Current Balance is : " +currentBalance);
```

Comment:
There is one requirement related this as well for individual user, that you missed here.
i want individual printTotalBalanace count as well.